src/pages/public_registration/RegistrationLandingPage.tsx

```diff
@@ -22,24 +24,48 @@ export const RegistrationLandingPage = () => {
   const { identifier } = useParams<IdentifierParams>();
   const user = useSelector((store: RootState) => store.user);
 
-  const registrationQuery = useQuery({
+  const registrationQuery = useQuery<Registration, AxiosError<DeletedRegistrationProblem>>({
```

Comment:
typen for dette settes implisitt nå, kanskje med unntatt error-typen, men den brukes uansett ikke lenger
```suggestion
  const registrationQuery = useQuery({
```